Samples~/SampleGame/README.md

```diff
@@ -1,147 +1,69 @@
 
-You can also use "GameLift / Configure Sample Server" and "GameLift / Configure Sample Client" in Unity main menu.
+## Integration with the deployment scenarios
 
-## How to deploy a server to GameLift
+The client will need the AWS Region, Cognito Client ID and API Gateway Endpoint which are stored in the scriptable object at `Assets\Settings\GameLiftClientSettings.asset`. When the scenario is deployed, these properties are imported automatically when clicking `Apply Managed EC2 Settings` from the `Host with Managed EC2` tab.
 
-Deploy a scenario to AWS from Unity with the plugin's GameLift/Deployment window. The fleet has to be in the active
-state before you can connect to it. You can check it at https://console.aws.amazon.com/gamelift/home (make sure you
-select your region on the page).
+The server code does not use these settings and integrates with the Amazon GameLift Server SDK instead of with the plugin.
 
-## Understandng and using the Client
+In this mode, the client can be run directly from the editor or as a standalone executable while the server is run by GameLift in the cloud.
 
-The client operates in three modes depending on what you are doing for a server.
+## Integration with Amazon GameLift Anywhere
 
-1. The game client will try to locate a local server, and if not found, attempt to find a GameLift server (with the
-   default alias or one supplied on the command line. If a server cannot be reached, the client will go into a
-   serverless standalone mode.
+The asset at `Assets\Settings\GameLiftClientSettings.asset` additionally stores a setting, `IsGameLiftAnywhere`, which generates setting YAML files when building the client and server executables.
 
-1. It will never again try to look for a server. This is by design. Just kill the client and start another if you want
-   another mode.
+The server settings are the parameters needed to call the [Amazon GameLift Server SDK API, InitSDK](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-server-sdk5-csharp-actions.html#integration-server-sdk5-csharp-initsdk-anywhere).
 
-"Hello World" appears in each client, and the clients enter a synchronized 'attract' mode (a color change animation),
-driven by the server.
+The server additionally requires an AuthToken generated from the [GetComputeAuthToken GameLift API](https://docs.aws.amazon.com/gamelift/latest/apireference/API_GetComputeAuthToken.html). When you run the server through the Unity editor, this token is generated and used automatically. If you run the server from a standalone executable, get a valid token and start the server from a command line, providing the auth token as an argument.
```

Comment:
Do you need to mention what that parameter looks like `--authToken`?

Reply:
Added